pyanaconda/modules/storage/partitioning/interactive/utils.py

```diff
@@ -94,6 +94,7 @@ def collect_unused_devices(storage):
         and not d.partitioned
         and (d.direct or d.isleaf)
         and d not in used_devices
+        and d.format.type != "iso9660"
```

Comment:
Is the purpose here to unmatch generally optical media? 

Maybe using find_optical_media is the correct way? 

@M4rtinK can you help with review please?

Reply:
I'm thinking that perhaps what we are seeing it actually a bug - if the media is used to start the installation, there should be code that puts it to the list of protected devices, which I would expect will prevent it from showing up at all. So maybe that is broken ?

BTW, it looks like iso9660 might be not just on optical media but also bootable installation flash drives, once you dd an installation image on them.

I guess there could be also the case of a user having two bootable installation flash drives connected, one used to boot to the installation and then another one just connected. Then the second one could show up as iso9660 volume & the question is how should Anaconda handle it ? I guess provide an option to delete the partition iso9660?